packages/webapp/pages/dao/index.tsx

```diff
@@ -174,3 +153,52 @@ export default function Proposals() {
     </DarkMode>
   )
 }
+
+export async function getServerSideProps() {
+  const block: Block | undefined = await getLatestBlock()
```

Comment:
might be good if this function indicated it's getting the latest ethereum block. ambiguous as it currently is

Reply:
👍